### ISSUE TITLE
Mac saving levels

### ISSUE THREAD

**Issue by Azusapink** [opentoonz/opentoonz#663]
i just updated to the new nightly build. When I save a level the image disappears and the name changes back to the default name. also the image i drew changes to what looks like a color palette. I might be crazy but someone please let me know if I am just saving my level wrong.

![screen shot 2016-07-20 at 10 37 38 pm](https://cloud.githubusercontent.com/assets/5882245/17010634/56940ad6-4ecb-11e6-8663-a60dafc17f6d.png)

**Comment by yamisama** [comment]
I don't think you are doing something wrong, this shouldn't be happening. It looks to me like some kind of issue with OpenGl contexts. I suspect that #659 may have caused this regression.

If you need to get production work done with OpenToonz, you can go back to an earlier nightly build for now until this problem is fixed. Earlier nightlies are available [here](https://github.com/opentoonz/opentoonz_nightlies/releases).

**Comment by Azusapink** [comment]
with the latest updates i get a blank level and the level text turns red
![screen shot 2016-08-07 at 4 38 46 pm](https://cloud.githubusercontent.com/assets/5882245/17465481/af721d64-5cbd-11e6-9e0c-baf07b4eb6ba.png)

**Comment by artisteacher** [comment]
@Azusapink, are you still having an issue saving levels? I have a Mac as well and usually don’t have a problem saving levels.

**Comment by RodneyBaker** [comment]
This issue looks VERY MUCH like the issue I have recently been having with raster levels:  For more information see:  https://github.com/opentoonz/opentoonz/issues/2117
